packages/exchange-io-erc1888/test/exchange.ts

```diff
@@ -0,0 +1,198 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Seems like this has been copied from exchange module, maybe better import or something rather than duplicating the code?

Reply:
I copied it and made slight changes, do not see a way to abstract